kitsune/search/api.py

```diff
@@ -36,60 +35,72 @@ def suggest(request):
     if errors:
         raise GenericAPIException(400, errors)
 
-    wiki_f = es_utils.F(
-        model='wiki_document',
-        document_category__in=settings.SEARCH_DEFAULT_CATEGORIES,
-        document_locale=locale,
-        document_is_archived=False)
+    searcher = (
+        es_utils.AnalyzerS()
+        .es(urls=settings.ES_URLS)
+        .indexes(es_utils.read_index('default')))
+
+    return Response({
+        'questions': _question_suggestions(searcher, text, locale, product, max_questions),
+        'documents': _document_suggestions(searcher, text, locale, product, max_documents),
+    })
+
 
-    questions_f = es_utils.F(
+def _question_suggestions(searcher, text, locale, product, max_results):
+    search_filter = es_utils.F(
```

Comment:
Bah--I just noticed this got merged already. I can do a PR with this change if you like.